entrypoint.sh

```diff
@@ -6,7 +6,8 @@ PORT="8000"
 set -e
 
 if [ "$1" = 'server' ]; then
-    gunicorn server:api -b :$PORT --chdir armada/api
```

Comment:
When using the sample config file as `armada.conf`, the API fails to start with the error `unable to load configuration from etc/armada/armada.conf`.